node.gyp

```diff
@@ -740,6 +740,9 @@
             }],
             [ 'node_use_perfctr=="false"', {
               'inputs': [ 'src/noperfctr_macros.py' ]
+            }],
+            [ 'node_lib_debug=="false"', {
+              'inputs': [ 'src/check_macros.py' ]
```

Comment:
Better name: nocheck_macro.py

That said, if you're using macros anyway, have two files: one with no-ops and one with, uh, ops, I guess?
```python
macro CHECK(x) = do { if (!(x)) throw new Error(`x`); } while (0);
```
For the no-ops, instead of evaluating to nothing, maybe emit something that is dead code but syntax-checked, to avoid bitrot:
```python
macro CHECK(x) = while (x, 0) {};
```
And a third suggestion is to have a guard variable so you can enable/disable checks at run-time instead of compile time.